frontend/src/lib/config.ts

```diff
@@ -3,6 +3,9 @@ import { KeyRound, Lock, Unlock, User } from "lucide-svelte";
 const config = {
   site: {
     name: "Cipherly",
+    description:
```

Comment:
Why does this config.ts file exist? And why are we moving various text that pertains to individual pages here?